src/web/features.rs

```diff
@@ -79,15 +80,22 @@ pub fn build_features_handler(req: &mut Request) -> IronResult<Response> {
 fn order_features_and_count_default_len(raw: Vec<Feature>) -> (Vec<Feature>, usize) {
     let mut feature_map = get_feature_map(raw);
     let mut features = get_tree_structure_from_default(&mut feature_map);
-    let mut remaining: Vec<_> = feature_map
+    let remaining: Vec<_> = feature_map
         .into_iter()
         .map(|(_, feature)| feature)
         .collect();
-    remaining.sort_by_key(|feature| feature.subfeatures.len());
 
     let default_len = features.len();
 
     features.extend(remaining.into_iter().rev());
+    features.sort_by(|a, b| {
+        // 'default' feature is special and should always stay on top
+        if b.name == "default" {
+            std::cmp::Ordering::Greater
+        } else {
+            a.name.partial_cmp(&b.name).unwrap()
+        }
+    });
```

Comment:
at first I've tried to simply change the sorting at the SQL level on [line 55](https://github.com/rust-lang/docs.rs/pull/1320/files#diff-0c19324803df42be471baabdc27c2f8f371245f7626c7534bfb17de43872dc5eR55) but I think this sorting in this function would neutralize it anyway